lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -9,7 +9,9 @@
  */
 export const BrowserConstants = {
     CACHE_LOCATION_LOCAL: "localStorage",
-    CACHE_LOCATION_SESSION: "sessionStorage"
+    CACHE_LOCATION_SESSION: "sessionStorage",
+    INTERACTION_STATUS_KEY: "interaction.status",
+    INTERACTION_IN_PROGRESS: "interaction_in_progress"
```

Comment:
the `_` for only one constant looks out of place while reading :)

Reply:
I understand, but most constants aren't going to be in this class, so I don't foresee this class growing past these variable. Happy to revisit later.

Reply:
okay!